packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -135,11 +135,11 @@ export interface ISemanticColors {
   //// Input controls slots (text fields, checkboxes, radios...)
 
   /**
-   * The border of an input control in its resting, unchecked state; e.g. the box of an unchecked checkbox.
+   * The border of a large input control in its resting, unchecked state; e.g. the box of dropdown.
```

Comment:
remove "unchecked" here, since larger input control don't usually have a checked state. add textbox to example list.